modules/manage/pages/cluster-maintenance/partition-recovery.adoc

```diff
@@ -0,0 +1,30 @@
+= Partition recovery
```

Comment:
Forced partition recovery.

IMO force signifies that it is not a regular raft recovery.

Reply:
I can change it, @mattschumpert any thoughts?